Bio/SeqIO/__init__.py

```diff
@@ -382,24 +382,24 @@
 from Bio.Align import MultipleSeqAlignment
 from Bio.Alphabet import Alphabet, AlphabetEncoder, _get_base_alphabet
 
-from . import AbiIO
-from . import AceIO
-from . import FastaIO
-from . import GckIO
-from . import IgIO  # IntelliGenetics or MASE format
-from . import InsdcIO  # EMBL and GenBank
-from . import NibIO
-from . import PdbIO
-from . import PhdIO
-from . import PirIO
-from . import SeqXmlIO
-from . import SffIO
-from . import SnapGeneIO
-from . import SwissIO
-from . import TabIO
-from . import QualityIO  # FastQ and qual files
-from . import UniprotIO
-from . import XdnaIO
+from Bio.SeqIO import AbiIO
```

Comment:
it is necessary to use the full path of imports to enable the doctests at the end of the file.